From a732c9118ebeffec1bfd7da11d5fd26915cd639e Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Wed, 16 Mar 2022 09:18:23 +0100
Subject: [PATCH] Fix KSyntaxHighlighting includes.

Mixing Camelcase and standard include wasn't the expected way. This was fixed in syntax-highlighting 5.91.
---
 src/common/Configuration.cpp     | 6 +++---
 src/common/SyntaxHighlighter.cpp | 2 +-
 src/common/SyntaxHighlighter.h   | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/common/Configuration.cpp b/src/common/Configuration.cpp
index a369f63..428f57f 100644
--- a/src/common/Configuration.cpp
+++ b/src/common/Configuration.cpp
@@ -7,9 +7,9 @@
 #include <QApplication>
 
 #ifdef IAITO_ENABLE_KSYNTAXHIGHLIGHTING
-#include <KSyntaxHighlighting/repository.h>
-#include <KSyntaxHighlighting/theme.h>
-#include <KSyntaxHighlighting/definition.h>
+#include <KSyntaxHighlighting/Repository>
+#include <KSyntaxHighlighting/Theme>
+#include <KSyntaxHighlighting/Definition>
 #endif
 
 #include "common/ColorThemeWorker.h"
diff --git a/src/common/SyntaxHighlighter.cpp b/src/common/SyntaxHighlighter.cpp
index 70f6ab0..8de3abd 100644
--- a/src/common/SyntaxHighlighter.cpp
+++ b/src/common/SyntaxHighlighter.cpp
@@ -5,7 +5,7 @@
 
 #include "Configuration.h"
 
-#include <KSyntaxHighlighting/theme.h>
+#include <KSyntaxHighlighting/Theme>
 
 SyntaxHighlighter::SyntaxHighlighter(QTextDocument *document) : KSyntaxHighlighting::SyntaxHighlighter(document)
 {
diff --git a/src/common/SyntaxHighlighter.h b/src/common/SyntaxHighlighter.h
index c5889e1..7f25d42 100644
--- a/src/common/SyntaxHighlighter.h
+++ b/src/common/SyntaxHighlighter.h
@@ -10,7 +10,7 @@
 
 #ifdef IAITO_ENABLE_KSYNTAXHIGHLIGHTING
 
-#include <KSyntaxHighlighting/syntaxhighlighter.h>
+#include <KSyntaxHighlighting/SyntaxHighlighter>
 
 class SyntaxHighlighter : public KSyntaxHighlighting::SyntaxHighlighter
 {
-- 
2.35.1

